test/e2e/utils.go

```diff
@@ -117,3 +154,12 @@ func nodeNetworkState(key types.NamespacedName) nmstatev1.NodeNetworkState {
 	}, timeout, interval).ShouldNot(HaveOccurred())
 	return state
 }
+
+func deleteNodeNeworkStates() {
+	nodeNetworkStateList := &nmstatev1.NodeNetworkStateList{}
+	err := framework.Global.Client.List(context.TODO(), &dynclient.ListOptions{}, nodeNetworkStateList)
+	Expect(err).ToNot(HaveOccurred())
+	for _, nodeNetworkState := range nodeNetworkStateList.Items {
+		framework.Global.Client.Delete(context.TODO(), &nodeNetworkState)
```

Comment:
I think we should also get the error here and configure a `Expect(err).ToNot(HaveOccurred())` because this can change the tests if we run the `AfterEach` and we are not able to remove all the objects before the next test.

Reply:
We want cleanup to continue, I will try to store a flag and check after loop.